merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/mocks/StubMissionModelService.java

```diff
@@ -17,7 +17,6 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.Optional;
```

Comment:
This was added in an earlier commit and should be removed there. I put a comment on it.